css/styles.css

```diff
@@ -173,8 +173,18 @@ img.notification-icon {
 
 .notification-container {
 	box-shadow: 0 1px 10px rgba(50, 50, 50, .7);
-	overflow: auto;
-	overflow-x: hidden;
+}
+
+/* Menu arrow */
+.notification-container:after {
+	right: 53px;
```

Comment:
This needs to be done in JS I think:

<img width="193" alt="bildschirmfoto 2017-01-26 um 12 46 48" src="https://cloud.githubusercontent.com/assets/245432/22345424/a6b5c27c-e3c5-11e6-8724-c4987ab9ff6c.png">
<img width="234" alt="bildschirmfoto 2017-01-26 um 12 47 01" src="https://cloud.githubusercontent.com/assets/245432/22345423/a6b46b20-e3c5-11e6-9443-f395ecb527e6.png">
<img width="268" alt="bildschirmfoto 2017-01-26 um 12 47 25" src="https://cloud.githubusercontent.com/assets/245432/22345425/a6b863f6-e3c5-11e6-9c98-79fd7486cfba.png">